### PR TITLE
Fix `asyncio.wait` code snippets

### PR DESCRIPTION
**Pull request by joseville1001**
Minor fix: Fixed `IndentationError: expected an indented block` by adding `print('yay!')`. Note: `print('yay!')` is what the `asyncio.wait_for` code snippet does, so I copied that behavior.

Real fix: Fixed `SyntaxError: 'await' outside async function` by putting most of the statements in their own coroutine, `main`, then executing that coroutine using `asyncio.run(main())`. I'm not sure if this is the proper fix, but it seems to illustrate the issue with passing a coroutine to `asyncio.wait` as the first code snippet doesn't print, but the second one does.

---

Re: the bpo issue number: I tried to create an issue using https://bugs.python.org/issue, but kept getting "Invalid csrf token found" when I trying to submit the issue.

Here is the issue title I used: "SyntaxError: 'await' outside function" in "asyncio-task.html#waiting-primitives" code snippets

Here is the issue comment I used: 

"""
https://docs.python.org/3/library/asyncio-task.html#asyncio.wait has the following two code snippets both of which fail with ""SyntaxError: 'await' outside function" when I run them in Python 3.9.7

Snippet 1:

```
async def foo():
    return 42

coro = foo()
done, pending = await asyncio.wait({coro})

if coro in done:
    # This branch will never be run!
    pass # I added this to prevent IndentationError
```

Snippet 2:

```
async def foo():
    return 42

task = asyncio.create_task(foo())
done, pending = await asyncio.wait({task})

if task in done:
    # Everything will work as expected now.
    pass # I added this to prevent IndentationError
```
"""

### CODE REVIEW

**Review by asvetlov — APPROVED** [review]

**Comment by asvetlov** [comment]
TLDR; Closing the PR, please make it against the `main` branch.

Hey!

The patch is correct.
Please make it against `main` branch, not `3.10`.
I am happy to review it again and merge but this PR should be closed by our development rules.

Also, please don't use `bpo-NNNN:` title to cheat our bots.
Almost every PR requires an issue on https://bugs.python.org
If a reviewer decides that the PR can relax this rule he/she can apply a GitHub label to skip a particular check. 
I think this particular case can avoid the bpo but I really appreciate it if you follow the full procedure.

**Comment by terryjreedy** [comment]
I have gotten the 'token' message.  I believe I fix it by starting a clean tab.

**Comment by joseville1001** [comment]
@terryjreedy and @asvetlov 
Thanks for the tip @terryjreedy. I'm still unable to create an issue in bugs.python.org. I tried doing so in a chrome incognito tab as well, but that also didn't work. Got:

`An error has occurred
A problem was encountered processing your request. The tracker maintainers have been notified of the problem.`

**Comment by gvanrossum** [comment]
Try a different browser?

**Comment by terryjreedy** [comment]
I occasionally get that and have to wait awhile, even a day.

**Comment by brettcannon** [comment]
We are also working on moving our issue tracker to GitHub, so at worst you could wait until that work is complete (hopefully in a couple of months at worst).
